iris-features.md

```diff
@@ -240,6 +412,33 @@ There are some new ID's that can be detected alongside items and armor:
 
 `trim_material` to detect armor trims on armor. (For example, `trim_emerald`).
 
+# Dimension Folders
+
+When `dimension.properties` is added to the shader pack root, behavior relating to dimensions change.
+Iris will no longer resolve any dimensions for you, and you are expected to resolve you own. The syntax for dimension.properties is as follows:
+
+`dimension.<folderName> = <dimensionNamespace>:<dimensionPath>`
+
+- `<folderName>`: The name of the folder containing the shaders used for the given dimension.
+- `<dimensionNamespace>`: The namespace for the dimension, vanilla shaders will use `minecraft`.
+- `<dimensionPath>`: The internal name of the dimension.
+
+*You can use `*` as a value to fallback all dimensions.*
+
+The following example sets the shaders for the vanilla Nether dimension to the `netherShaders` folder:
+
+`dimension.netherShaders = minecraft:the_nether`
+
+# Color Spaces (Iris 1.6.4)
+
+Iris 1.6.4 added support for additional color spaces beyond sRGB (DCI_P3, Display P3, REC2020, and Adobe RGB). This allows support for outputting to HDR displays.
```

Comment:
Iris does not allow output to HDR displays; this is just a tone mapping operation.

Reply:
Great, I probably should have played around with that a bit more before making that conclusion. Let me know if the change is accurate. Thanks for getting back so quickly!